app/views/layouts/_navbar.html.erb

```diff
@@ -55,7 +80,7 @@
           <%= link_to "Log In", login_path, class:'nav-link' %>
```

Comment:
Unable to login due to recaptcha

Reply:
Fixed this, the wrong method was being called in sessions controller. BUT if v3 has issues to validate, then the page renders again with V2 checkbox. If this becomes an issue, you can just render v2 directly by passing `show_recaptcha_v2:true`:

`<%= render_recaptcha('register', true %>`

Reply:
v3 does not work on login on my local, normal?